interfaces/IBF-dashboard/src/app/auth/auth.service.ts

```diff
@@ -112,7 +114,12 @@ export class AuthService implements OnDestroy {
     this.router.navigate(['/']);
   };
 
-  private onLoginError = (error) => {
+  private onLoginError = async ({ error }) => {
+    const toast = await this.toastController.create({
+      message: `Authentication Failed: ${error?.message}`,
```

Comment:
Is 'Unauthorized' clear enough? Or should we translate this to something like 'email and/or password unknown'? (so do that if error.status === 401, for other cases show error.message like now)